controllers/factory/scrapes.go

```diff
@@ -54,9 +68,24 @@ func CreateOrUpdateConfigurationSecret(ctx context.Context, cr *victoriametricsv
 		return nil, fmt.Errorf("selecting PodScrapes failed: %w", err)
 	}
 
-	ssCache, err := loadScrapeSecrets(ctx, rclient, smons, nodes, pmons, probes, statics, cr.Spec.APIServerConfig, cr.Spec.RemoteWrite, cr.Namespace)
+	ssCache, err := loadScrapeSecrets(ctx, rclient, sScrapes, nodes, pScrapes, probes, statics, cr.Spec.APIServerConfig, cr.Spec.RemoteWrite, cr.Namespace)
 	if err != nil {
-		return nil, fmt.Errorf("cannot load basic secrets for ServiceMonitors: %w", err)
+		if _, ok := err.(*utils.ErrGroup); !ok {
```

Comment:
I think it's can be too confusing, maybe it would be better to create specialized error for that (based on utils.ErrGroup) + check with errors.Is?

Reply:
Changed with errors.As